tests/unit/models/test_trophy.py

```diff
@@ -0,0 +1,22 @@
+"""Test praw.models.util."""
+from praw.models.trophy import Trophy
+import pytest
+
+from .. import UnitTest
+
+
+class TestTrophy(UnitTest):
+    def test_init__reject_no_name(self):
+        with pytest.raises(AssertionError):
+            Trophy(self.reddit, {})
+            Trophy(self.reddit, {'id': 'abcd'})
```

Comment:
Only one of these statements will actually raise right? If they both should, then shouldn't the second be in it's own `with pytest.raises....` block? If the first doesn't raise, then please take it out of the block.

Reply:
Thank you for pointing it out! Completely overlooked.